new/PRS14025Balca.xml

```diff
@@ -0,0 +1,75 @@
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Important military commander, royal treasurer and provincial governor. He participated in the battle of ʿAdwa and was killed
+                    by Italian forces during the Italian occupation in 1936.</p>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
+                <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+                <language ident="gez">Gəʿəz</language>
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="am" xml:id="n1">ባልቻ</persName>
+                    <persName xml:lang="am" type="normalized" corresp="#n1">Bālčā</persName>
+                    <persName xml:lang="am" xml:id="n2">ባልቻ፡ ሣፎ፡</persName>
+                    <persName xml:lang="am" type="normalized" corresp="#n2">Bālčā Śāfo</persName>
+                    <persName xml:lang="am" xml:id="n3">ደጃዝማች፡ ባልቻ፡</persName>
+                    <persName xml:lang="am" type="normalized" corresp="#n3">daǧǧāzmāč Bālčā</persName>
+                    <persName xml:lang="am" xml:id="n4">ደጃዝማች፡ ባልቻ፡ ሣፎ፡</persName>
+                    <persName xml:lang="am" type="normalized" corresp="#n4">daǧǧāzmāč Bālčā Śāfo</persName>
+                    <persName xml:lang="am" xml:id="n5" type="horse">አባ፡ ነፍሶ፡</persName>
+                    <persName xml:lang="am" type="normalized" corresp="#n5">Abbā Nafso</persName>
+                    <persName xml:lang="gez" xml:id="n6" type="baptismal">ተክለ፡ ሚካኤል፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n6">Takla Mikāʾel</persName>
+                    <t:faith xmlns:t="http://www.tei-c.org/ns/1.0" type="EOTC"/>
```

Comment:
t: and xmlns:t="http://www.tei-c.org/ns/1.0" have again snuck into the record (also in other place) - could you please delete them?